common/src/main/resources/less/style.less

```diff
@@ -523,12 +39,15 @@ i.icon.explore-button-icon:focus {
     0 0 0 0.2rem var(--focus-outer-color);
 }
 
+html {
+  // Required for iOS to work with 100vh
+  // https://css-tricks.com/css-fix-for-100vh-in-mobile-webkit/
+  height: -webkit-fill-avilable;
```

Comment:
I think this needs to be `-webkit-fill-available`. (an extra `a` in `available`)

Reply:
Oh thanks for noting it

Reply:
Maybe we should have a css linter as part of the build

Reply:
That would be nice. It's pretty easy for a typo to get missed.